src/index.js

```diff
@@ -63,34 +93,47 @@ class MaterielnetKonnector extends CookieKonnector {
         return new Promise((resolve, reject) => {
             logger.info("Retrieving login token");
             this.request(loginPageTokenOptions, (err, res) => {
+                if (err) {
+                    logger.debug(err.message);
+                    logger.error("Could not retrieve login token");
+                    return reject(new Error(errors.VENDOR_DOWN));
+                }
+                // Extract token
                 let token = "";
                 if (!err) {
                     let $ = cheerio.load(res.body);
                     token = $("#login form input[name='__RequestVerificationToken']").val();
                     if (!token)
                         err = new Error("No login token found");
                 }
-
-                if (err) {
-                    logger.debug(err.message);
-                    logger.error("Could not retrieve login token");
-                    return reject(new Error(errors.LOGIN_FAILED));
+                // Evaluate if a captcha is present
+                if (typeof(res.body) === "string" &&
+                    res.body.includes("window.renderCaptcha()")) {
+                    // We have encounter a captcha
+                    log("warn", "We detect a captcha");
```

Comment:
Could we use `logger.warn` to keep it consistent?

Reply:
Done